src/Nethermind/Nethermind.Test.Runner/blockchainTest1.json

```diff
@@ -0,0 +1,1072 @@
+{
```

Comment:
is this needed?

Reply:
Example blockchain test. useful in case i just want to copy paste and run inside VS